### PR TITLE
feat(tianmu): Improve the readbility of stonedb. (#11)

### PR DESCRIPTION
**Pull request by lylth**
[summary]
    1.class member variable refactor：common directory

## Summary about this PR

Issue Number: close #issue_number_you_created

## Tests Check List

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [x] New Feature
- [ ] Bug Fix
- [ ] Improvement
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: storage/tianmu/common/exception.h [hunk 1]
```diff
@@ -64,22 +64,22 @@ inline bool IsWarning(ErrorCode tianmu_rc) {
...
-    return error_messages[static_cast<int>(ec)];
+    if (!message_.empty())
+      return message_;
+    return error_messages[static_cast<int>(ec_)];
   }
 
  private:
-  ErrorCode ec;
-  std::string message;
+  ErrorCode ec_;
```
Comment (RingsC) [inline storage/tianmu/common/exception.h]: using `err_code_`, not just with underscore。 this pr main goal is to improve the code readibity. not just add some underscores.

File: storage/tianmu/common/data_format.h [hunk 2]
```diff
@@ -43,7 +43,7 @@ enum class EDF { TRI_UNKNOWN };
 
 class DataFormat {
  protected:
-  DataFormat(std::string name, EDF edf) : name(name), id(no_formats++), edf(edf) {}
+  DataFormat(std::string name, EDF edf) : name(name), id(no_formats_++), edf(edf) {}
```
Comment (RingsC) [inline storage/tianmu/common/data_format.h]: no_format is `no forma`t or `num_of_format`.

File: storage/tianmu/common/data_format.cpp [hunk 3]
```diff
@@ -27,26 +27,26 @@
...
-int DataFormat::no_formats = 0;
+int DataFormat::no_formats_ = 0;
 
-std::map<std::string, DataFormatPtr> DataFormat::df_map = {{"EDF::TRI_UNKNOWN", std::make_shared<TxtDataFormat>()}};
+std::map<std::string, DataFormatPtr> DataFormat::df_map_ = {{"EDF::TRI_UNKNOWN", std::make_shared<TxtDataFormat>()}};
 
 DataFormatPtr DataFormat::GetDataFormat(const std::string &name) {
-  auto it = df_map.find(boost::trim_copy(boost::to_upper_copy(name)));
-  return it != df_map.end() ? it->second : DataFormatPtr();
+  auto it = df_map_.find(boost::trim_copy(boost::to_upper_copy(name)));
```
Comment (RingsC) [inline storage/tianmu/common/data_format.cpp]: `df_map` is a map for what? the readers can understand the basic idea of the code via the literal words.

**Review by RingsC — CHANGES_REQUESTED** [review]

**Review by hustjieke — APPROVED** [review]
LGTM

**Review by RingsC — APPROVED** [review]

**Pull request closed without merging**